src/github/handlers/index.ts

```diff
@@ -46,8 +44,19 @@ async function handleEvent(event: EmitterWebhookEvent, eventHandler: InstanceTyp
 
   for (const pluginChain of pluginChains) {
     if (pluginChain.skipBotEvents && "sender" in event.payload && event.payload.sender?.type === "Bot") {
+      console.log("Skipping plugin chain because sender is a bot");
       continue;
     }
+    if (
+      context.key === "issue_comment.created" &&
+      pluginChain.command &&
+      "comment" in context.payload &&
+      !context.payload.comment.body.startsWith(pluginChain.command)
+    ) {
+      console.log("Skipping plugin chain because command does not match");
```

Comment:
I think the command should be defined in the config and not within the plugin. This approach doesn't seem right.

Reply:
It is defined in the config

Reply:
This doesn't make sense what is this logic supposed to be doing

Reply:
For example config:
```
issue_comment.created:
  - name: "plugin A"
    command: "/a"
    uses: ...
  - name: "plugin B"
    command: "/b"
    uses: ...
```
When `issue_comment` is triggered the kernel will check if comment body starts with `/a` and skip plugin A if doesn't, it will do the same for plugin B.